docs/content/how-to/kubevirt/create-kubevirt-cluster.md

```diff
@@ -53,31 +86,59 @@ export PULL_SECRET="$HOME/pull-secret"
 export MEM="6Gi"
 export CPU="2"
 export WORKER_COUNT="2"
+export BASE_DOMAIN=hypershift.lab
+# OCP_RELEASE is optional, this will determine which version is used for the Hosted Cluster
+# if not set, it will default to the hardcoded version in the hypershift cli
+export OCP_RELEASE=4.12.7-x86_64
 
 hypershift create cluster kubevirt \
 --name $CLUSTER_NAME \
---node-pool-replicas=$WORKER_COUNT \
+--node-pool-replicas $WORKER_COUNT \
 --pull-secret $PULL_SECRET \
 --memory $MEM \
---cores $CPU
+--cores $CPU \
+#--release-image quay.io/openshift-release-dev/ocp-release:$OCP_RELEASE \
```

Comment:
i'd rather omit the release-image from the example command. we want hypershift cli to pick the best default for us. if someone is using the latest hypershift operator, the default will be accurate

Reply:
I will remove it and add a note stating that this flag could be used to provision the cluster with a specific OCP release. Let me know if this is good enough or if we want to completely omit this information.

From my point of view I see it useful in terms of testing, let's say that you're running a hypershift version that supports deploying from 4.11 to 4.13 hosted clusters. Being able to specify a given release may be helpful when troubleshooting issues / reproducing a bug / testing something in a specific release / etc.